test/decl/class/actor/basic.swift

```diff
@@ -53,3 +53,5 @@ extension A2 {
 
   init(doesNotDelegate: ()) {} // expected-error {{designated initializer cannot be declared in an extension of 'A2'}}
 }
+
+nonisolated actor A3 {} // expected-warning {{'nonisolated' modifier cannot be applied to this declaration; this will be an error in a future Swift language mode}}{{1-13=}}
```

Comment:
@ktoso just to confirm that i'm not off by one here... does the range here seem correct? i wasn't actually sure how to test the fixit 'for real'